tests/backup/backup_helper.go

```diff
@@ -1445,3 +1445,187 @@ func DeleteBackupAndWait(backupName string, ctx context.Context) error {
 	_, err := task.DoRetryWithTimeout(backupDeletionSuccessCheck, backupDeleteTimeout, backupDeleteRetryTime)
 	return err
 }
+
+func CreateBackupUsingNamespaceLabels(backupName string, clusterName string, bLocation string, bLocationUID string, labelSelectors map[string]string, orgID string, uid string, preRuleName string,
```

Comment:
Please refer to PR #1231. It might be a redundant function.

Reply:
Done